apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts

```diff
@@ -0,0 +1,248 @@
+      ].getText()
+      await expect(deviceSerialNumber).toEqual(device.serialNumber)
+    }
+  })
+
+  it("Connect 3rd Kompakt device", async () => {
+    E2EMockClient.mockResponse({
+      path: "path-3",
+      body: overviewDataWithOneSimCard3rd,
+      endpoint: "FEATURE_DATA",
+      method: "GET",
+      status: 200,
+    })
+    E2EMockClient.addDevice({
+      path: "path-3",
+      serialNumber: thirdSerialNumber,
+    })
+
+    await browser.pause(6000)
+    const menuItem = await $(`//a[@href="#/generic/mc-overview"]`)
+
+    await menuItem.waitForDisplayed({ timeout: 10000 })
+    await expect(menuItem).toBeDisplayed()
+  })
+
+  it("Verify Device 3 on the Drawer and check the Serial Number", async () => {
+    const deviceSelectDrawer = await drawerPage.deviceSelectDrawer
+    await expect(deviceSelectDrawer).toBeDisplayed()
+
+    const deviceImageOnDrawer = await drawerPage.deviceImageOnDrawer
+    await expect(deviceImageOnDrawer).toBeDisplayed()
+
+    const thirdDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      thirdSerialNumber
+    )
+
+    await expect(thirdDeviceOnDrawer).toBeDisplayed()
+
+    const drawerSerialNumbers = drawerPage.drawerDeviceSerialNumber
+
+    const thirdDrawerDeviceSerialNumber = await drawerSerialNumbers[2].getText()
+    await expect(thirdDrawerDeviceSerialNumber).toEqual(thirdSerialNumber)
+  })
+
+  it("Connect 4,5,6 Kompakt devices", async () => {
+    const devices = [
+      {
+        path: "path-4",
+        body: overviewDataWithOneSimCard4th,
+        serialNumber: fourthSerialNumber,
+      },
+      {
+        path: "path-5",
+        body: overviewDataWithOneSimCard5th,
+        serialNumber: fifthSerialNumber,
+      },
+      {
+        path: "path-6",
+        body: overviewDataWithOneSimCard6th,
+        serialNumber: sixthSerialNumber,
+      },
+    ]
+
+    for (const device of devices) {
+      E2EMockClient.mockResponse({
+        path: device.path,
+        body: device.body,
+        endpoint: "FEATURE_DATA",
+        method: "GET",
+        status: 200,
+      })
+      E2EMockClient.addDevice({
+        path: device.path,
+        serialNumber: device.serialNumber,
+      })
+
+      await browser.pause(6000)
+    }
+  })
+
+  it("Verify Devices 4,5 and 6 on Drawer and check their Serial Numbers, scroll down and select 6th Kompakt", async () => {
+    const deviceSelectDrawer = await drawerPage.deviceSelectDrawer
+    await expect(deviceSelectDrawer).toBeDisplayed()
+
+    const deviceImageOnDrawer = await drawerPage.deviceImageOnDrawer
+    await expect(deviceImageOnDrawer).toBeDisplayed()
+
+    const fourthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      fourthSerialNumber
+    )
+    const fifthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      fifthSerialNumber
+    )
+    const sixthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      sixthSerialNumber
+    )
+    await expect(fourthDeviceOnDrawer).toBeDisplayed()
```

Comment:
🔴 mockdev.51
```
Error: Expect $(`[data-testid="device-select-drawer-content"]`) to be displayed

Expected: "displayed"
Received: "not displayed"
```

Reply:
[chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] » \src\specs\stress-tests\device-drawer-stress-test.ts
[chrome 116.0.5845.228 windows #0-0] Kompakt switching devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 1,2 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 1 and 2 on Drawer and check their Serial Numbers
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 3rd Kompakt device
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Device 3 on the Drawer and check the Serial Number
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 4,5,6 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 4,5 and 6 on Drawer and check their Serial Numbers, scroll down and select 6th Kompakt
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify 6th Kompakt with new Serial Number and click Show connected devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Disconnect the devices and check if Welcome Page is present
[chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] 8 passing (42.7s)


Spec Files:      1 passed, 1 total (100% completed) in 00:00:45

mockdev 51 ok